R/ets.R

```diff
@@ -220,7 +220,7 @@ specials_ets <- new_specials(
 #' \subsection{season}{
 #' The `season` special is used to specify the form of the seasonal term and associated parameters.
 #' \preformatted{
-#' season(method = c("N", "A", "M"), period = NULL,
+#' season(method, period = NULL,
```

Comment:
These should match the signature of the specials function. No changes necessary here.
It may be worth updating the `method` explanation to state that if multiple methods are specified, the best one for the data will be chosen automatically.